tests/channels/test_y_to_d0_d0bar_pi0_pi0.py

```diff
@@ -21,8 +21,9 @@ def test_simple(formalism_type, n_solutions, particle_database):
         number_of_threads=1,
     )
     assert len(result.transitions) == n_solutions
-    model = es.generate_amplitudes(result)
-    assert len(model.parameters) == 9
+    model_builder = es.amplitude.get_builder(result)
+    model = model_builder.generate()
+    assert len(model.parameters) == 4
```

Comment:
why did this change from 9 to 4? Complex amplitude coefficients right?

Reply:
Yes, meson radius for the top decay plus 4 coefficients. These were the parameters previously (produced by running `io.write` in this test for the master branch).
![image](https://user-images.githubusercontent.com/29308176/110809943-a080f800-8285-11eb-85c4-b1270d2b0d29.png)